File: courses/CS50/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

//int main(void)
//{
//    int times;
//    do
//    {
//        times = get_int("Height: ");
//    }
//    while (times < 1 || times > 8);
//
//    if (times == 1)
//    {
//        printf("#\n");
//    }
//    else
//    {
//        for (int i = 1; i < times + 1; i++)
//        {
//            for (int j = 0; j < times - i; j++)
//            {
//                printf(" ");
//            }
//            for (int k = 0; k < i; k++)
//            {
//                printf("#");
//            }
//            printf("\n");
//        }
//    }
//}

int main(void)
{
    int times;
    do
    {
        times = get_int("Height: ");
    }
    while (times < 1 || times > 8);
    if (times == 1)
    {
        printf("#  #\n");
    }
    else
    {
        for (int i = 1; i < times + 1; i++)
        {
            for (int j = 0; j < times - i; j++)
            {
                printf(" ");
            }
            for (int k = 0; k < i; k++)
            {
                printf("#");
            }
            printf("  ");

            for (int n = 0; n < i; n++)
            {
                printf("#");
            }
            printf("\n");
        }
    }
}

File: courses/CS50/pset6/mario/less/mario.py
#lines = int(input("Height: "))
#for i in range(1, lines + 1):
#    for j in range(lines - 1, 0, -1):
#        print(" " * j, end="")
#    print("#")

from cs50 import get_int

while True:
    lines = get_int("Height: ")
    if lines > 0 and lines < 9:
        break
j = 1

for i in range(lines - 1, -1, -1):
    print(" " * i, end ="")
    print("#" * j)
    j += 1
